cli/azd/internal/appdetect/spring_boot.go

```diff
@@ -379,6 +381,25 @@ func IsValidDatabaseName(name string) bool {
 	return re.MatchString(name)
 }
 
+func detectPropertySpringCloudStreamBindingDestination(azdProject *Project, springBootProject *SpringBootProject) {
+	result := getBindingDestinationMap(springBootProject.applicationProperties)
+	if len(result) != 0 {
+		azdProject.Metadata.BindingDestinationInProperty = result
+	}
+}
+
+func detectPropertyEventhubsCheckpointStoreContainer(azdProject *Project, springBootProject *SpringBootProject) {
+	result := make(map[string]string)
```

Comment:
It's it necessary to use a map to store the information? It's just a string value:
> ![image](https://github.com/user-attachments/assets/c0720ca6-d0d4-4f80-ad54-09bc8c20871d)

Reply:
What you show is the single binder case, it does have only one.
However, in the multi-binders scenario, it may specify multiple containers, and each have different key, as follows.

https://github.com/azure-javaee/azure-spring-boot-samples/blob/main/eventhubs/spring-cloud-azure-stream-binder-eventhubs/eventhubs-multibinders/spring3sample/src/main/resources/application.yaml
![image](https://github.com/user-attachments/assets/a9cad7fb-389e-4842-80ce-bb75175e6e71)